src/config.rs

```diff
@@ -244,7 +254,19 @@ impl From<cli::Opt> for Config {
             tab_width: opt.tab_width,
             tokenization_regex,
             true_color: opt.computed.true_color,
-            truncation_symbol: "→".to_string(),
+            truncation_symbol: format!("{}→{}", ansi::ANSI_SGR_REVERSE, ansi::ANSI_SGR_RESET),
+            wrap_config: side_by_side_wrap::WrapConfig {
+                wrap_symbol: opt.side_by_side_wrap_symbol,
+                wrap_right_symbol: opt.side_by_side_wrap_right_symbol,
+                right_align_symbol: opt.side_by_side_wrap_right_align_symbol,
+                // TODO, support multi-character symbols, and thus store
+                // right_align_symbol_len here?
+                use_wrap_right_permille: {
+                    let percent = opt.side_by_side_wrap_right_percent.clamp(0.0, 100.0);
```

Comment:
I wanted to use `clamp` but wouldn't it mean changing the delta build environment to use nightly rust? I thought that that might cause problems with CI/CD and package manager build systems.
```
error[E0658]: use of unstable library feature 'clamp'
```

Reply:
`clamp()` became stable with 1.50, or do you want to target a specific rustc version?

Reply:
Ah sorry, I was confused, my local compiler just needed updating.